modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -79,6 +79,23 @@ public int getWeight() {
 		return _weight;
 	}
 
+	public boolean hasParameterTypes(
+		String content, String[] parameterList, String[] parameterTypes) {
```

Comment:
Can we rename `content` to `methodCall` or `methodCallContent` to make it clear that it is meant for method call statements?

Reply:
But Kevin, it's meant to be the `content` of the file. why do you say it would be `methodcall`?

Reply:
Wait, but doesn't that mean that this method needs to take __four__ parameters instead of __three__?:

```java
public boolean hasParameterTypes(
		String methodCall, String content, String[] parameterList, String[] parameterTypes);
```